spec/dummy/app/controllers/application_controller.rb

```diff
@@ -8,4 +8,22 @@ def show
       render text: "", layout: "application"
     end
   end
+
+  def static_endpoint
+    if params[:use_current_user]
+      respond_raw_text("Current user is now #{current_user}")
+    else
+      respond_raw_text("static text")
+    end
+  end
+
+  private
+
+  def respond_raw_text(text)
```

Comment:
Can we respond with `head :ok` here and avoid rendering text at all? Doesn't look like the tests require it.

Reply:
yes, no need for that indeed, will change